assets/development.services.yml

```diff
@@ -0,0 +1,13 @@
+# Local development services.
+#
+# To activate this feature, follow the instructions at the top of the
+# 'example.settings.local.php' file, which sits next to this file.
+parameters:
```

Comment:
@radimklaska could we rely on the development.services.yml provided by the drupal core instead of us keeping it here.

Reply:
Source: https://git.drupalcode.org/project/drupal/-/tree/10.0.x/core/assets/scaffold/files